DESCRIPTION

```diff
@@ -24,9 +24,9 @@ Imports:
     tidyr, 
     ttservice (>= 0.3.8),
     SummarizedExperiment,
+    tidySummarizedExperiment,
```

Comment:
tidySCE should not depend on tidySE, the only dependency should be with the mother package ttservice.